src/token-tracker-link.ts

```diff
@@ -4,10 +4,16 @@ export = function getTokenTrackerLink(
   tokenAddress: string,
   network: string,
   holderAddress?: string,
+  customNetworkUrl?: string,
 ): string {
+  if(customNetworkUrl?.length){
+    return `${customNetworkUrl}/token/${tokenAddress}${ 
+      holderAddress ? `?a=${ holderAddress }`: ''}`;
```

Comment:
The last part of this (`?a=${ holderAddress }`) isn't part of the EIP. Maybe it's best that we left this out for now.

For custom networks we can just use the account link instead, and rely upon the user to find their token transactions from there.